integration-tests/src/test/groovy/com/okta/sdk/tests/it/AuthorizationServerIT.groovy

```diff
@@ -136,6 +137,7 @@ class AuthorizationServerIT extends ITSupport {
                 .setDescription("Test Authorization Server")
                 .setAudiences(["api://example"]))
         registerForCleanup(createdAuthorizationServer)
+        assertThat(createdAuthorizationServer, notNullValue())
```

Comment:
Remove this duplicate line. The below line already tests this.

Reply:
done